packages/elements/src/utils/validators/validate-require.ts

```diff
@@ -10,7 +10,9 @@ export const validateRequire = <ValuesType, ErrorKeys extends string>({
   return validateBase<ValuesType, ErrorKeys>({
     values,
     keys,
-    validator: str => str !== '',
+    validator: str => {
+      return typeof str === 'string' ? str.trim() !== '' : str !== ''
```

Comment:
I think need to change to look more cleaner

Reply:
I updated